fuel_test/clear_env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eu
+
+. localrc
+
+[ $DEBUG == "on" ] && set -x
+
+function clear_xs {
+	local xs_host="$1"
+	ssh -qo StrictHostKeyChecking=no root@$xs_host \
+	'
+	set -eux
+
+	route del -net 192.168.0.0 gw 169.254.0.2 netmask 255.255.255.0 dev xenapi
+	route del -net 192.168.1.0 gw 169.254.0.2 netmask 255.255.255.0 dev xenapi
+	> /etc/sysconfig/static-routes
+	rm /root/rotate_xen_guest_logs.sh
+
+	yum remove openstack-neutron-xen-plugins.noarch openstack-xen-plugins.noarch -y
```

Comment:
If check the existing firstly, as the CI may terminate before they are installed.